### PR TITLE
Add GSoC contributor post

### PR DESCRIPTION
**Pull request by leewujung**
This PR will add the first post from our GSoC contributor @ldr426.

@ldr426: once we settle on the text, you could PR to my forked-branch https://github.com/leewujung/group-website/tree/gsoc-post and i'll merge it and your post will make its way to the main site via this PR.

### CODE REVIEW

File: content/post/2022-07-28-gsoc-ldr426-intro/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+After determining which type of tool to use, I began to read a user guide about HoloViz libraries. There are several libraries mainly used in echoshader: hvplot, Holoviews, GeoViews and Panel.  Hvplot and HoloViews declare objects for instantly visualizable data, building Bokeh plots from convenient high-level specifications. GeoViews visualize geographic data corresponding to ship survey datasets. Panel assembles grams and control widgets from these different libraries into a layout which could be displayed in a Jupyter notebook and in a standalone servable dashboard.  In addition to HoloViz libraries, PyVista and other libraries are involved for 3D extension, which also fit well in panel layout. Also, benchmarking and doc work are required for each module.
+
```
Comment (leewujung) [inline content/post/2022-07-28-gsoc-ldr426-intro/index.md:51]: Below are some screenshots of the different visualization functionalities I am developing:

2d_echogram

tracks

curtain

Comment (leewujung) [inline content/post/2022-07-28-gsoc-ldr426-intro/index.md:51]:
```suggestion
Below are some screenshots of the different visualization functionalities I am developing:

![2d_echogram](https://user-images.githubusercontent.com/15334215/186999651-76081a29-11f8-4d37-b3a9-fca0ad49a03c.png)

![tracks](https://user-images.githubusercontent.com/15334215/186999662-ba744a49-b02e-4451-a716-f8c8df654053.png)

![curtain](https://user-images.githubusercontent.com/15334215/186999678-2bf77985-aab3-42f8-88f9-1f2c78d3b2eb.png)
```

**Comment by leewujung** [comment]
@lsetiawan : I made up a title and put in a short paragraph before Dingrui's text, see what you think.

File: content/post/2022-07-28-gsoc-ldr426-intro/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+title: "Hello from Dingrui Lei, GSoC contributor on Echoshader!"
```
Comment (lsetiawan) [inline content/post/2022-07-28-gsoc-ldr426-intro/index.md:3]:
```suggestion
title: "Hello from Dingrui Lei, GSoC contributor of Echoshader!"
```

File: content/post/2022-07-28-gsoc-ldr426-intro/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+date: 2022-07-28T00:00:00-08:00
+authors: 
+  - Dingrui Lei
+  - landungs
```
Comment (lsetiawan) [inline content/post/2022-07-28-gsoc-ldr426-intro/index.md:7]: Are these suppose to be github handles? if so please change to `ldr426` and `lsetiawan`

**Review by lsetiawan — APPROVED** [review]
This is great. Thanks for doing this. Just some small detail that may make it sound better, and also a simple question regarding authors. Otherwise, I like your additional intro.

File: content/post/2022-07-28-gsoc-ldr426-intro/index.md (hunk 3 above)
Comment (leewujung) [reply content/post/2022-07-28-gsoc-ldr426-intro/index.md:7]: No, this renders to be author list. lsetiwan links to your profile automatically at the end of post, Dingrui doesn't have a profile so it's just his name.

Comment (lsetiawan) [reply content/post/2022-07-28-gsoc-ldr426-intro/index.md:7]: Ah I see. Cool. Thanks for letting me know 😄

**Comment by leewujung** [comment]
Thanks reviewing @lsetiawan ! I'll merge this now.

**Pull request merged**